remotehandler.py

```
@@ -0,0 +1,294 @@
+import boto3
+import time
+from botocore.exceptions import ClientError
+from botocore.exceptions import NoCredentialsError
+
+S3_BUCKET = 'ictrl-test2024'
+EXTENSION = '.clp.zst'
+
+
+# Used for local file MD5 hash key calculation
+def calculate_hash(file_path):
+    hash = hashlib.md5()
```

Ruff (0.8.2)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)